libobs/obs.h

```diff
@@ -1324,6 +1324,8 @@ EXPORT void obs_source_set_monitoring_type(obs_source_t *source,
 					   enum obs_monitoring_type type);
 EXPORT enum obs_monitoring_type
 obs_source_get_monitoring_type(const obs_source_t *source);
+EXPORT void obs_source_set_monitor_enabled(obs_source_t *source, bool enable);
+EXPORT bool obs_source_monitor_enabled(const obs_source_t *source);
```

Comment:
The documentation related to those functions is missing.
Please add it in this PR.

Reply:
Done